app/javascript/constants/writeOffActFields.ts

```diff
@@ -29,15 +29,18 @@ export const writeOffActTableCell = [
     title: 'Lost quantity',
   },
   {
-    id: 'bundle_seria',
-    title: 'Bundle series',
+    id: 'description',
+    title: 'Description',
   },
+];
+
+export const writeOffActTableCell = [
   {
-    id: 'bundle_number',
-    title: 'Bundle number',
+    id: 'consignment',
```

Comment:
![image](https://user-images.githubusercontent.com/70818761/167442446-fdd741fb-f17b-4e67-90cf-5b84be176170.png)
i guess its just a typo?

Reply:
No, but I know, that need to fix.